services/explorer/src/utils/networks.ts

```diff
@@ -116,3 +116,22 @@ export const networkRedirect = (network: string): void => {
     window.location.replace(redirectUrl);
   }
 };
+
+export const recreateNetworkIndexerConnection = (currentNetwork: string) => {
+  let indexerEndpoint: string;
+  switch (currentNetwork) {
+    case Network.MAINNET:
+      indexerEndpoint = process.env.REACT_APP_MAINNET_INDEXER_HOST || '';
+      break;
+    case Network.TESTNET:
+      indexerEndpoint = process.env.REACT_APP_TESTNET_INDEXER_HOST || '';
+      break;
+    case Network.DEVNET:
+      indexerEndpoint = process.env.REACT_APP_DEVNET_INDEXER_HOST || '';
+      break;
+    default:
+      indexerEndpoint = process.env.REACT_APP_MAINNET_INDEXER_HOST || '';
+      break;
+  }
+  return indexerEndpoint;
+};
```

Comment:
I don't think we need this. We already have a connection to the indexer that we use in the nodes / address pages

Reply:
yes, this has been cleaned up in [this](https://github.com/Taraxa-project/taraxa-platform/pull/542/commits/e2d1a642969ef10f7ac7e9e56938d670e4e812aa) commit, we had a little pair programming session with Florin on it

Reply:
So this will be removed once we merge the event logs one too

Reply:
After we resolve all of these comments, let's merge that PR in here and check again